scratch/fetch_info.py

```
@@ -0,0 +1,91 @@
+  "recommendations": detailed_items,
+  "searchContext": {
+    "todayOverview": "本日は幅広いカテゴリーで特選タイムセールや注目商品が多数展開されており、日常使いからエンタメまで多彩な商品がお得になっています。",
+    "searchedCategories": list(seen_categories)
+  }
+}
+
+with open("data/recommendations/today.json", "w") as f:
+    json.dump(output, f, ensure_ascii=False, indent=2)
+
+print("Done! Validating...")
+subprocess.run(["uv", "run", "python", "scripts/validate_artifact.py", "data/recommendations/today.json"])
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)